packages/ai/cypress/specs/AINoticeIndicator.cy.tsx

```diff
@@ -0,0 +1,167 @@
+import AINoticeIndicator from "../../src/AINoticeIndicator.js";
+
+describe("AI Notice Indicator", () => {
+	describe("Rendering and Interaction", () => {
+		beforeEach(() => {
+			cy.mount(<AINoticeIndicator></AINoticeIndicator>);
+		});
+
+		const attributionText = "Created with AI.";
+		const verificationText = "Verify results before use.";
+
+		it('Should render the component', () => {
+			cy.get('ui5-ai-notice-indicator').should('exist');
+		});
+
+		it("Should display closed responsive poppover", () => {
+			cy.get("[ui5-ai-notice-indicator]")
+				.shadow()
+				.find("[ui5-responsive-popover]")
+				.should("exist");
+
+			cy.get("[ui5-ai-notice-indicator]")
+				.shadow()
+				.find("[ui5-responsive-popover]")
+				.should("have.prop", "open", false);
+		});
+
+		it("Should display correct properties for Default mode", () => {
+
+			cy.get("[ui5-ai-notice-indicator]")
+				.shadow()
+				.find("[ui5-link]")
+				.should('have.attr', 'icon', '')
+				.should("exist");
+
+			cy.get("[ui5-ai-notice-indicator]")
+				.shadow()
+				.find("[ui5-link]")
+				.contains(attributionText)
```

Comment:
Adjust these assertions to match the new approach described in the other comments (having defaults from i18n if property is not set.)
(Not just here, but everywhere that is applicable)

In the tests for what is rendered in the different modes just test what shows up in the dom.
Add separate tests for actually checking the values of the texts - one for checking default values, one for custom set ones.